src/pages/activity/landingPage/LandingPage.tsx

```diff
@@ -13,7 +13,7 @@ const LandingPage = () => {
         <div className="py-7 md:pt-[145px] md:pb-[125px] flex justify-between flex-col-reverse md:flex-row">
           <div className="w-full md:w-[400px]">
             <Slide
-              data={productsData.products.iphone13.mainImgs}
+              data={productsData.products["iphone 13"].mainImgs}
```

Comment:
please change the mainImgs data as a key of products, instead of type.

Reply:
yes sir